osgar/tools/lidarview.py

```diff
@@ -195,10 +195,18 @@ def get_image(data):
     return image
 
 
+def draw_bbox(image, bbox):
+    assert len(bbox) == 5, bbox
+    name, x, y, width, height = bbox
+    color = (255, 0, 0)
```

Comment:
display in green instead
```suggestion
    color = (0, 255, 0)
```